### PR TITLE
[12.0][MIG] Base partner sequence

### PR DESCRIPTION
**Pull request by CasVissers-360ERP**
Migration of base_partner_sequence to v12

@astirpe Can you help me with my first OCA PR?

### CODE REVIEW

File: base_partner_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+{
+    "author": "Tiny/initOS GmbH & Co. KG,"
+              "ACSONE SA/NV,"
+              "Tecnativa, "
+              "Odoo Community Association (OCA), "
+              "Camptocamp, ",
+    "name": "Add a sequence on customers' code",
+    "version": "12.0.1.0.0",
+    "category": "Generic Modules/Base",
+    "website": "http://www.initos.com",
```
Comment (nikul-serpentcs) [inline base_partner_sequence/__manifest__.py]: @CasVissers `website` URL replace as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: base_partner_sequence/__init__.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Copyright 2004-2009 Tiny SPRL (<http://tiny.be>).
```
Comment (nikul-serpentcs) [inline base_partner_sequence/__init__.py]: @CasVissers Remove Copyright from `init` file

File: base_partner_sequence/tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# Copyright 2004-2009 Tiny SPRL (<http://tiny.be>).
```
Comment (nikul-serpentcs) [inline base_partner_sequence/tests/__init__.py:2]: same here

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: base_partner_sequence/tests/__init__.py (hunk 3 above)
Comment (CasVissers-360ERP) [reply base_partner_sequence/tests/__init__.py:2]: No need to do it in the manifest?

**Comment by gdgellatly** [comment]
@CasVissers you need to follow the migration guide for preserving commit history. When we review, normally we review the entire file, but then just the final commit so we can see changes.

Which means following this guide - https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

If I were you, I'd start in a new branch, then rebase/cherrypick your 3 commits as one squashed commit on top of it, plus sort out the README (after rebasing etc).  Sorting out the README involves the following 3 steps - at least how I do it.

First you need to clone OCA/maintainer-tools - in my case that is `~/devtools/maintainer-tools` and pip install it, preferably into a virtualenv.  Instructions in README for repo

1. Create files: 
cd to the module directory and I have a simple fish function, but you can just use this line
```bash
cp -r ~/devtools/maintainer-tools/template/module/readme .
```
2. Edit these files. 
Basically delete their existing content and copy over relevant section from existing README making edits as required. Delete unused sections.
3. Generate README.
  EDIT: cd to the repo directory e.g. partner-contact
  Activate virtualenv.
  Run `oca-gen-addon-readme --repo-name partner-contact --branch 12.0 --addon-dir base_partner_sequence`

Assuming you committed and squashed your migration work before then `git add --all`, `git commit --amend --no-edit` and `git push -f`.

**Comment by sbidoul** [comment]
This seems stalled, superseded by #668
